scripts/performance.js

```diff
@@ -4,91 +4,91 @@
  */
 
 function stamp(message, time = new Date() - performance.timing.navigationStart, type = '') {
-    // eslint-disable-next-line no-console
```

Comment:
Did you actually change anything in there?

Reply:
I think all these changes are coming from the main branch of bamboohr. I merged latest `main` into my PR